packages/select/package.json

```diff
@@ -36,7 +36,7 @@
   "peerDependencies": {
     "@availity/api-axios": "^3.0.1",
     "@availity/api-core": "^3.0.0",
-    "@availity/form": "^0.2.0",
+    "@availity/form": "^0.2.0 || ^0.5.34",
```

Comment:
Same thing about versions in between 0.2.0 and 0.5.0. As well as, 0.5.34 -> 0.5.0